src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CodeActionEndpoint.cs

```diff
@@ -186,13 +181,13 @@ private async Task<ImmutableArray<RazorVSInternalCodeAction>> GetDelegatedCodeAc
         // No point delegating if we're in a Razor context
         if (languageKind == RazorLanguageKind.Razor)
         {
-            return ImmutableArray<RazorVSInternalCodeAction>.Empty;
+            return [];
```

Comment:
>[] [](http://example.com/codeflow?start=19&length=2)

Is this doing the same exact thing as before (using a single cached empty ImmutableArray object) or is it creating a new one every time?